etc/singularity.conf

```diff
@@ -141,3 +163,14 @@ container dir = /var/singularity/mnt
 # better option.
 #sessiondir prefix = /var/singularity/sessions/
 
+
+# SINGULARITY USER: [STRING]
+# DEFAULT: singularity
+# The protected image mode will restrict Singularity images to those owned by
+# the 'singularity user'; this option allows admins to change the username we
+# use as a test.
+#
+# If user namespaces are enabled, the singularity user setting is ignored and
```

Comment:
`singularity` is the user that must own the image file (when utilizing protected image mode).  The test performed is literally whether `singularity` can `fopen()` the file (and whether it owns the image file).  I don't foresee admins changing the name of the user, but I dislike hardcoding usernames (what if the site already has a user named `singularity`, used for other purposes?).

Because we perform the checks as the `singularity` user (instead of looking at the mode / ownership and "recreating" POSIX), we:
* Are less likely to introduce bugs as we are using the Linux kernel's logic.
* Will function in the face of with less-common access control methods such as file attributes and ACLs.

Since there is the `group` mode that provides more fine-grained permissions, I don't think there's much use in allowing the admin to specify multiple `singularity` users.